### PR TITLE
t1

### PR DESCRIPTION
**Pull request by asifnaseebi**

### CODE REVIEW

File: utils/reader.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import os
+
+def file_read():
+
```
Comment (Hamma111) [inline utils/reader.py:4]: too many new lines, looking strange. correct this

File: utils/reader.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    with open("/home/asif/Desktop/project/barq_weather_task/files/f1.csv") as f:
+        content = f.read()
+    f.close()
+
+    rows = []
+
+    for row in content.split("\n")[1:-1]:
+        rows.append(row)
+    return rows
```
Comment (Hamma111) [inline utils/reader.py:13]: before return statement there should always be a new line

File: utils/reader.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+file_read()
```
Comment (Hamma111) [inline utils/reader.py:15]: no need to call this function here. remove it

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t1.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from utils.reader import file_read
+files_values = file_read()
```
[the remark this answers is not in the document]
Comment (asifnaseebi) [reply task/t1.py:2]: ok

File: utils/reader.py (hunk 3 above)
Comment (asifnaseebi) [reply utils/reader.py:15]: ok sir

**Review by Hamma111 — APPROVED** [review]

**Pull request merged**